File: 程式語言 C V2答案檔/108.c
#include <stdio.h>
#include <stdlib.h>

int main ()
{
    const double pi = 3.1415;
    int d;
	double r,area;

	scanf("%d", &d);
    r = d / 2.0;
	area = r * r * pi;
	
	int r_2 = (r+0.005)*100;
	int area_2 = (area+0.00005)*10000;

    printf("%-10d\n", d);
    printf("%-10.2f\n", r_2/100.);
	printf("%-10.4f\n", area_2/10000.);

 	return 0;
}

File: 程式語言 V2答案檔/C#/108.cs
using System;

public class TQC108
{
    public static void Main(string[] args)
    {
        const double pi = 3.1415;
        int d;
        double r, area;
        
        d = int.Parse(Console.ReadLine());
        r = d / 2.0;
        area =  r * r * pi;

        double r_2 = ((int)((r + 0.005) * 100)) / 100.0;
        double area_2 = ((int)((area + 0.00005) * 10000)) / 10000.0;

        Console.WriteLine(String.Format("{0, -10}", d));
        Console.WriteLine(String.Format("{0, -10:F2}", r_2));
        Console.WriteLine(String.Format("{0, -10:F4}", area_2));
    }
}